qadence/backends/pyqtorch/convert_ops.py

```diff
@@ -222,165 +244,161 @@ def sparse_operation(state: Tensor, values: dict[str, Tensor] = None) -> Tensor:
                 convert_block(block, n_qubits, config),
             )
 
-        if config.use_gradient_checkpointing:
-
-            def _forward(state: Tensor, values: dict[str, Tensor] = None) -> Tensor:
-                new_state = checkpoint(self.operation, state, values, use_reentrant=False)
-                return pyq.overlap(state, new_state)
-
-        else:
-
-            def _forward(state: Tensor, values: dict[str, Tensor] = None) -> Tensor:
-                return pyq.overlap(state, self.operation(state, values))
-
-        self._forward = _forward
-
     def forward(self, state: Tensor, values: dict[str, Tensor]) -> Tensor:
-        return self._forward(state, values)
+        return pyq.overlap(state, self.operation(state, values))
+
+    def run(self, state: Tensor, values: dict[str, Tensor]) -> Tensor:
+        return self.operation(state, values)
 
 
 class PyQHamiltonianEvolution(Module):
     def __init__(
         self,
-        qubit_support: Sequence,
+        qubit_support: Tuple[int, ...],
```

Comment:
`tuple[int, ...]` to maintain consistency.

Reply:
mysterious github shows changes i didnt make. its still tuple[int,...]

Reply:
ah it was sequence and i changed it to tuple. you read it wrong